input_source.m

```diff
@@ -5,7 +5,7 @@
 static NSString *const kSourceID =
     @"im.rime.inputmethod.Squirrel";
 static NSString *const kInputModeID =
-    @"im.rime.inputmethod.Squirrel.Rime";
+    @"im.rime.inputmethod.Squirrel.Rime.Hans";
```

Comment:
This ID is different from any tsInputModeListKey defined in Info.plist.

Reply:
Fixed as follows:
im.rime.inputmethod.Squirrel.Rime(.*) => im.rime.inputmethod.Squirrel(.*)